metricflow-semantics/metricflow_semantics/model/semantics/semantic_model_lookup.py

```diff
@@ -67,6 +67,9 @@ def __init__(self, model: SemanticManifest, custom_granularities: Dict[str, Expa
         # Cache for defined time granularity.
         self._time_dimension_to_defined_time_granularity: Dict[TimeDimensionReference, TimeGranularity] = {}
 
+        # Cache for agg. time dimension for measure.
```

Comment:
did you mean to include a period after agg?

Reply:
Yes, since it is an abbreviation.